app/models/katello/content_override.rb

```diff
@@ -0,0 +1,36 @@
+module Katello
```

Comment:
There isn't anything about 'models' that requires them to be tied to active record.  Reading some debate online it seems to really be up to each project to decide their own 'convention'.